frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -195,9 +204,23 @@ const LogsTabForPodName: React.FC<{ podName: string; isFailedPod: boolean }> = (
                       <ToolbarItem spacer={{ default: 'spacerSm' }} style={{ maxWidth: '200px' }}>
                         <SimpleDropdownSelect
                           isDisabled={podContainers.length <= 1}
-                          options={podContainers.map((container) => ({
-                            key: container.name,
-                            label: container.name,
+                          options={podStepStates.map((podStepState) => ({
+                            key: podStepState.stepName,
+                            label: podStepState.stepName,
+                            dropdownLabel: (
+                              <>
+                                {podStepState.stepName}{' '}
+                                {podStepState.state === 'Error' ? (
+                                  <Icon status="danger">
+                                    <ExclamationCircleIcon />
+                                  </Icon>
+                                ) : (
+                                  <Icon status="success">
+                                    <CheckCircleIcon />
+                                  </Icon>
```

Comment:
does this not also have a loading state?

Reply:
if it is loading it shows as success

Reply:
There can only be a loading state when the logs of that particular step are loading.

Reply:
Currently, if there is an "error" in the logs of individual steps, I'm returning "Error" else "Success"

Reply:
For example, i click on a step's pod and it is shown that it is loading b y the spinner at the top of the logs, but in the dropdown it shows success. The confusion i am having is that your success state is also your loading state.

Reply:
added.